Sensus.Android/AndroidSensusServiceHelper.cs

```diff
@@ -833,5 +829,15 @@ public override void BringToForeground()
         }
 
         #endregion
+
+        public void StopAnroidSensusService()
+        {
+            _service.Stop();
+        }
+
+        public SensorManager GetSensorManager()
+        {
+            throw new NotImplementedException();
```

Comment:
Oh yeah. Whoops. Nice catch. Just as an FYI, this is mostly a stop gap change. Once we've managed to work out a lot of the dependencies and move AndroidSensusServiceHelper into the shared android project this won't be necessary.